components/shared_code/tests/shared_data_model/meta/test_parameter.py

```diff
@@ -14,33 +17,58 @@ class ParameterTest(MetaModelTestCase):
 
     def test_check_help_and_help_url(self):
         """Test that a parameter can only have one of help and help URL."""
-        expected_error = "Parameter Parameter has both help and help_url"
-        self.check_validation_error(expected_error, name="Parameter", help="Help.", help_url="https://help.example.org")
+        self.check_validation_error(
+            "Parameter Parameter has both help and help_url",
+            name="Parameter",
+            type=ParameterType.STRING,
+            help="Help.",
+            help_url=HttpUrl("https://help.example.org"),
+            metrics=["loc"],
+        )
+        self.check_validation_error(
+            "The help string of parameter Parameter does not end with a dot",
```

Comment:
Coincidentally, errors do not have a full stop either :D

Reply:
Not necessarily. The check method does a `error_text in pydantic_output` check ;-)